packages/reactive-element/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lit/reactive-element",
-  "version": "1.4.1",
+  "version": "1.4.2",
```

Comment:
Oh, actually there should be a corresponding bump to lit for this

Reply:
Good catch! Thank you!! Last time I re-ran the release workflow, the force push that updates the PR made the workflow wonky with no way to run and merge, so I'll close this PR and create a new one.